lib/customer.rb

```diff
@@ -0,0 +1,28 @@
+require 'twilio-ruby'
+
+module Customer
+
+  attr_reader :send_text
+
+ def send_text
+    # put your own credentials here
+    account_sid = 'ACdff620df7376a7affc0d0ae499f860d1'
+    auth_token = 'e2db70e02bb0b68a9994bd0893d5a875'
+
```

Comment:
Account credentials are now publicly available to anyone who wants to impersonate you on twilio! These should be extracted to environment variables.